AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/Model/KeyStoreAdmin.smithy

```diff
@@ -42,16 +42,16 @@ service KeyStoreAdmin {
     CreateKey,
     VersionKey,
     InitializeMutation,
-    ApplyMutation,
-    ResumeMutation
+    ApplyMutation
+    // ResumeMutation,
+    // DescribeMutation
```

Comment:
?

Reply:
Long story.
We need them to IA.
We thought, under guidance from @seebees , 
that we would not.

But our better angles defeated our inner demons,
and we need these.